cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c

```diff
@@ -20,7 +20,7 @@
 #define LOG_MODULE_NAME mcumgr_flash_mgmt
 #define LOG_LEVEL CONFIG_IMG_MANAGER_LOG_LEVEL
 #include <logging/log.h>
-LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+LOG_MODULE_REGISTER(zephyr_img_mgmt);
```

Comment:
Change LOG_MODULE_NAME instead?

Reply:
It doesn't work :) It prints out as LOG_MODULE_NAME

Reply:
Oh!

So perhaps nuke first two lines, and just do `LOG_MODULE_REGISTER(zephyr_img_mgmt, CONFIG_IMG_MANAGER_LOG_LEVEL);` ?